Automattic/pocket-casts-ios
# [Winback] Add FF and first screen skeleton

| 📘 Part of: #2435
|:---:|

Fixes #2436 

This PR adds:
- A new FF called `winback`
- The skeleton of the new screen

| Default | FF on |
| -------- | ------- |
| ![RocketSim_Screenshot_iPhone_16_6 1_2024-11-18_11 34 35](https://github.com/user-attachments/assets/4fe5ff64-2bd3-4df2-9ebc-403dc64846e7) | ![RocketSim_Screenshot_iPhone_16_6 1_2024-11-18_11 34 56](https://github.com/user-attachments/assets/df463e96-3d5f-4b31-9b9d-f67762363c70) |

## To test

- CI must be 🟢 
- Run the app with a plus/patron user
- If you run the app with a non plus user, enable plus from the developer panel
- Go to Account and tap `Cancel Subscription`
- You should see the `Cancel Subscription` view
- Go back and enable the FF `winback`
- Navigate again to Account and tap `Cancel Subscription`
- You should see the skeleton view (empty view with a label in the middle like the screenshot above)

## Checklist
